packages/flutter/lib/src/foundation/consolidate_response.dart

```diff
@@ -43,13 +44,13 @@ typedef BytesReceivedCallback = void Function(int cumulative, int total);
 /// bytes from this method (assuming the response is sending compressed bytes),
 /// set both [HttpClient.autoUncompress] to false and the `autoUncompress`
 /// parameter to false.
-Future<Uint8List> consolidateHttpClientResponseBytes(
+Future<TransferableTypedData> consolidateHttpClientResponseBytes(
```

Comment:
Technically this looks like a breaking change, but I'll let you judge whether it's worth sending an announcement.